src/web/mjs/engine/Storage.mjs

```diff
@@ -869,4 +876,37 @@ export default class Storage {
         }
         return undefined;
     }
+
+    /**
+     * Save the given value for the given key in the bookmark storage
+     */
+    saveBookmarks( key, value, indentation ) {
+        return new Promise( ( resolve, reject ) => {
+            this.fs.writeFile( this.path.join(Storage._bookmarkOutputPath.value, 'hakuneko.' + key), JSON.stringify( value, undefined, indentation ), function(error ) {
```

Comment:
The getter is not static, so it needs to be accessed via instance (`this`), also the getter already returns the `value` property, simply `this._bookmarkOutputPath` will do the trick